lib/connection_pool/timed_stack.rb

```diff
@@ -118,6 +129,27 @@ def length
     @max - @created + @que.length
   end
 
+  ##
+  # Pre-create all possible connections
+  def fill
```

Comment:
It's actually a fairly common feature request upstream that they've been rejecting for years -- the desire to pre-fill the pool if connection initialization is expensive and you want to do it at startup instead of lazily. I don't personally have a use for it, but I figured I'd add it while I was in here.